ocs_ci/ocs/ripsaw.py

```diff
@@ -88,6 +88,26 @@ def apply_crd(self, crd):
         run(f'oc apply -f {crd}', shell=True, check=True, cwd=self.dir)
         run(f'oc apply -f {self.operator}', shell=True, check=True, cwd=self.dir)
 
+    def get_uuid(self, benchmark):
+        """
+        Getting the UUID of the test.
+           when ripsaw used for running a benchmark tests, each run get its own
+           UUID, so the results in the elastic-search server can be sorted.
+
+        Args:
+            benchmark (str): the name of the main pod in the test
+
+        Return:
+            str: the UUID of the test
```

Comment:
Add an empty line before the end of the docstring

```suggestion
            str: the UUID of the test

```

Reply:
add